hydra-node/json-schemas/logs.yaml

```diff
@@ -720,6 +720,22 @@ definitions:
               "$ref": "api.yaml#/components/schemas/Party"
             description: >-
               Parties configured as the node argument.
+      - title: SavedNetworkPartiesInconsistent
+        description: >-
+          The configured peer list does not match with the value from the loaded /acks state.
```

Comment:
It is a bit surprising to see the concrete path `/acks` here. IMO it has a high chance to bitrot here.

Should: not be so specific here

---

If the intention was to explain what to do in case you see this error in the logs, I think this would have not been discovered anyways by anyone (logs.yaml is not really part of our documentation).